src/core/points.rkt

```diff
@@ -24,7 +24,7 @@
 ;; points; and 2) a ground-truth output for each input.
 
 (define *pcontext* (make-parameter #f))
-(struct pcontext (points exacts))
+(struct pcontext (points exacts) #:prefab)
```

Comment:
I guess this is fine.

Reply:
Ya I was worried about this when I found out `(place-message-allowed? v)` pretty much requires it on any structs being passed around.

Reply:
Yeah I am increasingly feeling like there's no point to having internal data structures since we eventually always need a `→json` version anyway. Might as well pass around JSON-able data to begin with.

Reply:
Just more historical baggage in Racket and most other languages.